package.json

```diff
@@ -18,14 +18,16 @@
   },
   "scripts": {
     "watch": "yarn install-if-deps-outdated && ./node_modules/.bin/gulp watch",
-    "clean-client": "rm -rf _inc/build/ _inc/blocks/ css/",
+    "clean": "yarn clean-client && yarn clean-extensions",
+    "clean-client": "rm -rf _inc/build/ css/",
+    "clean-extensions": "rm -rf _inc/blocks/ ",
```

Comment:
Is the reasoning to have `clean-client` and `clean-extensions` separate because also build commands are separate?